README.md

```diff
@@ -35,6 +35,12 @@ To list URLs to repositories of packages that belong to network component:
 znoyder browse-osp packages --component network --tag osp-17.0 --output osp-patches
 ```
 
+There is a argument used in the output to filter project name:
```

Comment:
Maybe better: There is an additional key `osp-project` introduced that contains downstream repository name
?

Reply:
You're right. It's a better explanation.

Change done.